src/context/auth-context.tsx

```diff
@@ -32,6 +32,11 @@ export const AuthDataProvider = ({ children }: AuthDataProviderProps) => {
         [activeLoginid, accountsList]
     );
 
+    const URLParams = new URLSearchParams(window.location.search);
+    const authURLParams = !!URLParams.get('acct1') || !!URLParams.get('token1');
```

Comment:
There is an edge case when the user is redirected from the OAuth page to the Deriv platform's page. In this scenario, `!!Cookies.get('authToken') && !isAuthorized` was returning `true` because there was a cookie with an empty object as the value, and `isAuthorized` was not completed yet. Therefore, I need to account for this condition in my checks to ensure the correct value for `isAuthorizing` is always returned.

Reply:
@shayan-deriv, did you test `authorized subscriptions`? do they still work as expected?